### PR TITLE
Redesign of Register form, adding email and arrival time to form and …

### PR DESCRIPTION
**Pull request by everamal**
…model

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "codelyzer": "^5.0.0",
     "jasmine-core": "^3.4.0",
     "jasmine-spec-reporter": "~4.2.1",
-    "karma": "~4.1.0",
```
Comment (Miguelsiqueiros) [inline package.json]: Revert this change, the rest looks good.

Comment (alfavenegas) [reply package.json]: x2

Comment (everamal) [reply package.json]: done!
